src/frontend/src/lib/types/qr-code.ts

```diff
@@ -6,3 +6,28 @@ export type QrResponse = {
 	token?: string;
 	amount?: number;
 };
+
+type DecodedUrnBase = {
+	prefix: string;
+	destination: string;
+	networkId?: string;
```

Comment:
We might confuse `networkId: string` with `networkId: NetworkId`, would it be worth using a slightly different name?